README.md

```diff
@@ -8,7 +8,7 @@ Image available from:
 * [**Docker Hub**](https://hub.docker.com/r/galexrt/healthchecks)
 
 
-Simple to use Docker image for [https://github.com/healthchecks/healthchecks](healthchecks/healthchecks).
+Simple to use Docker image for [https://github.com/healthchecks/healthchecks](https://github.com/healthchecks/healthchecks).
```

Comment:
Could you please also change the name for the name (in `[]`) to just `github.com/healthchecks/healthchecks` (without https)?

Reply:
Like this?